virt-v2v/cold/entrypoint.go

```diff
@@ -49,7 +49,7 @@ func main() {
 	fmt.Println("Preparing virt-v2v")
 	switch source {
 	case vSphere:
-		virtV2vArgs = append(virtV2vArgs, "-i", "libvirt", "-ic", os.Getenv("V2V_libvirtURL"))
+		virtV2vArgs = append(virtV2vArgs, "--root=first", "-i", "libvirt", "-ic", os.Getenv("V2V_libvirtURL"))
```

Comment:
shouldn't it be
```suggestion
		virtV2vArgs = append(virtV2vArgs, "--root first", "-i", "libvirt", "-ic", os.Getenv("V2V_libvirtURL"))
```

Reply:
I'm pretty sure it will work, but can you please run a real migration with this change? (the CI doesn't test a real conversion)